### PR TITLE
Enable updating links to previous and current release in vaem metadata

### LINKED ISSUE #1018: vaem links to latest/previous versions are not updated during build

### PR DESCRIPTION
**Pull request by fkleedorfer**
Fixes #1018

### CODE REVIEW

**Review by steveraysteveray — COMMENTED** [review]
I will hold off until you resolve the merge conflicts in pom.xml

**Comment by fkleedorfer** [comment]
> I will hold off until you resolve the merge conflicts in pom.xml

Fair enough. coming right up

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**
